main.js

```diff
@@ -0,0 +1,76 @@
+let bookList = [];
```

Comment:
- [ ] Kindly add a default collection that has some books when first visiting the page.